.codesandbox/tasks.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
are `.codesandbox/tasks.json` and `.devcontainer` needed for dependabot?

Reply:
No.

Reply:
Can you remove the extra files?